openeo/extra/job_management.py

```diff
@@ -437,15 +471,25 @@ def _update_statuses(self, df: pd.DataFrame):
                 con = self._get_connection(backend_name)
                 the_job = con.job(job_id)
                 job_metadata = the_job.describe()
-                _log.info(
-                    f"Status of job {job_id!r} (on backend {backend_name}) is {job_metadata['status']!r}"
-                )
-                if job_metadata["status"] == "finished":
-                    self.on_job_done(the_job, df.loc[i])
+                _log.info(f"Status of job {job_id!r} (on backend {backend_name}) is {job_metadata['status']!r}")
+
+                if (df.loc[i, "status"] == "created" or df.loc[i, "status"] == "queued" or df.loc[i, "status"] == "started") and job_metadata["status"] == "running":
+                    df.loc[i, "running_time"] = datetime.now(timezone.utc).strftime('%Y-%m-%dT%H:%M:%S.%fZ') 
```

Comment:
I agree on the confusing. 

The idea was to remain consistent with the start_time field in pandas. Ideally I would go for start_running_timestamp, but I think this is confusing as well in relation to start_time.

Reply:
maybe running_state_timestamp?

It could be usefull to revise start_time to a similar notation; however this is not backwards compatible

Reply:
I don't think it's necessary to include `status` (or `state`) in the name. I would also stick to `time`, like in existing `start_time`, instead of introducing `timestamp` 

So `start_running_time` or `running_start_time`. Maybe even `start_running` or `running_start` would work :smile:

Reply:
sure, however I want to avoid using rfc3999.utcnow as it creates a string. For the comparisson I actually need a timedelta object. 

I will take a look if this is also an option through the rfc package